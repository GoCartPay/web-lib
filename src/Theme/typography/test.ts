// import { typography } from './index'

// // Note that if the design token changes a used value, the snapshot will need to be updated
// it('Produces correct output', () => {
//   expect(typography).toMatchSnapshot()
// })
