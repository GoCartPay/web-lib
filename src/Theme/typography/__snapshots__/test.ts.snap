// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Produces correct output 1`] = `
Object {
  "body1": Object {
    "fontFamily": "TWK Lausanne",
    "fontSize": 16,
    "fontStretch": "normal",
    "fontStyle": "normal",
    "fontWeight": 400,
    "letterSpacing": 0,
    "lineHeight": 21,
    "paragraphIndent": 0,
    "paragraphSpacing": 0,
    "textCase": "none",
    "textDecoration": "none",
  },
  "body2": Object {
    "fontFamily": "TWK Lausanne",
    "fontSize": 14,
    "fontStretch": "normal",
    "fontStyle": "normal",
    "fontWeight": 400,
    "letterSpacing": 0,
    "lineHeight": 19,
    "paragraphIndent": 0,
    "paragraphSpacing": 0,
    "textCase": "none",
    "textDecoration": "none",
  },
  "caption": Object {
    "fontFamily": "TWK Lausanne",
    "fontSize": 12,
    "fontStretch": "normal",
    "fontStyle": "normal",
    "fontWeight": 400,
    "letterSpacing": 0.4,
    "lineHeight": 19.92,
    "paragraphIndent": 0,
    "paragraphSpacing": 0,
    "textCase": "none",
    "textDecoration": "none",
  },
  "fontFamily": "TWK Lausanne",
  "h1": Object {
    "fontFamily": "TWK Lausanne",
    "fontSize": 64,
    "fontStretch": "normal",
    "fontStyle": "normal",
    "fontWeight": 400,
    "letterSpacing": -2,
    "lineHeight": 73,
    "paragraphIndent": 0,
    "paragraphSpacing": 0,
    "textCase": "none",
    "textDecoration": "none",
  },
  "h2": Object {
    "fontFamily": "TWK Lausanne",
    "fontSize": 48,
    "fontStretch": "normal",
    "fontStyle": "normal",
    "fontWeight": 400,
    "letterSpacing": -2,
    "lineHeight": 54,
    "paragraphIndent": 0,
    "paragraphSpacing": 0,
    "textCase": "none",
    "textDecoration": "none",
  },
  "h3": Object {
    "fontFamily": "TWK Lausanne",
    "fontSize": 36,
    "fontStretch": "normal",
    "fontStyle": "normal",
    "fontWeight": 400,
    "letterSpacing": -1.8,
    "lineHeight": 40,
    "paragraphIndent": 0,
    "paragraphSpacing": 0,
    "textCase": "none",
    "textDecoration": "none",
  },
  "h4": Object {
    "fontFamily": "TWK Lausanne",
    "fontSize": 30,
    "fontStretch": "normal",
    "fontStyle": "normal",
    "fontWeight": 400,
    "letterSpacing": -1.8,
    "lineHeight": 36,
    "paragraphIndent": 0,
    "paragraphSpacing": 0,
    "textCase": "none",
    "textDecoration": "none",
  },
  "h5": Object {
    "fontFamily": "TWK Lausanne",
    "fontSize": 24,
    "fontStretch": "normal",
    "fontStyle": "normal",
    "fontWeight": 400,
    "letterSpacing": 0,
    "lineHeight": 28,
    "paragraphIndent": 0,
    "paragraphSpacing": 0,
    "textCase": "none",
    "textDecoration": "none",
  },
  "h6": Object {
    "fontFamily": "TWK Lausanne",
    "fontSize": 20,
    "fontStretch": "normal",
    "fontStyle": "normal",
    "fontWeight": 400,
    "letterSpacing": 0,
    "lineHeight": 25,
    "paragraphIndent": 0,
    "paragraphSpacing": 0,
    "textCase": "none",
    "textDecoration": "none",
  },
  "legal": Object {
    "fontFamily": "TWK Lausanne",
    "fontSize": 9,
    "fontStretch": "normal",
    "fontStyle": "normal",
    "fontWeight": 400,
    "letterSpacing": 0,
    "lineHeight": 14,
    "paragraphIndent": 0,
    "paragraphSpacing": 0,
    "textCase": "none",
    "textDecoration": "none",
  },
  "legal_url": Object {
    "fontFamily": "TWK Lausanne",
    "fontSize": 9,
    "fontStretch": "normal",
    "fontStyle": "normal",
    "fontWeight": 400,
    "letterSpacing": 0,
    "lineHeight": 14,
    "paragraphIndent": 0,
    "paragraphSpacing": 0,
    "textCase": "none",
    "textDecoration": "underline",
  },
  "overline": Object {
    "fontFamily": "TWK Lausanne",
    "fontSize": 12,
    "fontStretch": "normal",
    "fontStyle": "normal",
    "fontWeight": 400,
    "letterSpacing": 1.2,
    "lineHeight": 16,
    "paragraphIndent": 0,
    "paragraphSpacing": 0,
    "textCase": "uppercase",
    "textDecoration": "none",
  },
  "subtitle1": Object {
    "fontFamily": "TWK Lausanne",
    "fontSize": 16,
    "fontStretch": "normal",
    "fontStyle": "normal",
    "fontWeight": 400,
    "letterSpacing": 0,
    "lineHeight": 21,
    "paragraphIndent": 0,
    "paragraphSpacing": 0,
    "textCase": "none",
    "textDecoration": "none",
  },
  "subtitle2": Object {
    "fontFamily": "TWK Lausanne",
    "fontSize": 14,
    "fontStretch": "normal",
    "fontStyle": "normal",
    "fontWeight": 400,
    "letterSpacing": 0,
    "lineHeight": 19,
    "paragraphIndent": 0,
    "paragraphSpacing": 0,
    "textCase": "none",
    "textDecoration": "none",
  },
  "title": Object {
    "fontFamily": "TWK Lausanne",
    "fontSize": 96,
    "fontStretch": "normal",
    "fontStyle": "normal",
    "fontWeight": 400,
    "letterSpacing": -2,
    "lineHeight": 108,
    "paragraphIndent": 0,
    "paragraphSpacing": 0,
    "textCase": "none",
    "textDecoration": "none",
  },
}
`;
